web-common/src/features/sources/workspace/SourceWorkspaceHeader.svelte

```diff
@@ -166,18 +172,37 @@
   );
   $: isSourceUnsaved = $isSourceUnsavedQuery.data;
 
-  $: reconciliationErrors = getFileArtifactReconciliationErrors(
-    $fileArtifactsStore,
-    `${sourceName}.yaml`
-  );
+  const handleCreateModelFromSource = async () => {
+    const modelName = await createModelFromSourceV2(queryClient, sourceName);
+    goto(`/model/${modelName}`);
```

Comment:
Do we not need to redirect to modeller if there is an error in creating the model?

Reply:
What are you suggesting? If `createModelFromSourceV2` throws an error, the `goto` won't get hit.

Reply:
Ya that is what i was thinking. But this should probably be handled when we integrate the new reconcile. Any `put` success should navigate to the page, reconcile errors will be filled asynchronously by the new reconcile.